cluster_test.go

```diff
@@ -22,22 +23,12 @@ var clusterSlotKeys = func() [numSlots]string {
 			ksmall := k[:i]
 			if a[ClusterSlot(ksmall)] == "" {
 				a[ClusterSlot(ksmall)] = string(ksmall)
+				found++
 				break
 			}
 		}
-
-		var notFull bool
-		for _, k := range a {
```

Comment:
Turns out iterating over up to 16k strings takes some time when you do this 20,000 times (which is the average number of iterations for the outer loop on my machine)